### PR TITLE
Preserve timestamps

### PR DESCRIPTION
**Pull request by schnittstabil**
This PR adds a new option to preserve timestamps (atime and mtime), even though the **File timestamp resolution is inconsistent with fs.stat / fs.utimes: joyent/node#7000**.

I'm just a bit uncertain about the default value, actually it's `true`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cp-file",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (sindresorhus) [inline package.json]: don't bump verions other peoples projects ;)

**Comment by sindresorhus** [comment]
Does it really need an option though? At least on OS X the default behaviour is preserving created/modified time when copying a file.

File: package.json (hunk 1 above)
Comment (schnittstabil) [inline package.json]: Sorry

**Comment by schnittstabil** [comment]
At least on Ubuntu 14.04 the timestamps are not synced.

**Comment by schnittstabil** [comment]
Afaik, there are filesystems w/o mtime support - but I don't know how node would handle them...

**Comment by sindresorhus** [comment]
> Or do you mean cp-file should always preserve the timestamps?

Yes.

> Afaik, there are filesystems w/o mtime support - but I don't know how node handles them - so the possibility to disable the preservation seems reasonable to me.

Pretty sure Node handles that gracefully.

> Anyway, the Linux cp command allows one to disable the preservation, same holds for the copy command on Windows.

I get that kind of arguments a lot, and it does make sense, but consider that most of those tools are iterated for years and have a lot of baggage. I prefer to only add stuff when there's an apparent and valid use-case for it.

**Comment by schnittstabil** [comment]
> I prefer to only add stuff when there's an apparent and valid use-case for it.

:+1:

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by schnittstabil** [comment]
Sweet :) - Which anime is this?

**Comment by sindresorhus** [comment]
No idea. Just found it somewhere.

**Pull request merged**